### DISCUSSION TITLE
New version of QGrid is released

### DISCUSSION THREAD

**Discussion by pratik227** [quasarframework/quasar#13159] [Show and Tell]
Hello Guys, Just released a new version of QGrid. You can access it using the following URL.

Github - https://github.com/pratik227/quasar-qgrid

Doc link - https://next-quasar-qgrid.netlify.app

New features Added
1. Date range filter
2. Select all option is available now
3. Draggable columns support is added

**Comment by metalsadman** [comment]
awesome work.

**Reply by pratik227** [reply]
Thanks @metalsadman 😄

**Comment by jandresmedina** [comment]
@pratik227  Hi nice to meet you,

 Is there a way to export only the filtered data in Q-grid, if so, could you tell me how I can do it from the front or at the source code level of your extension, thanks greetings
